button.html

```diff
@@ -0,0 +1,323 @@
+
+                                    <i class="fa fa-trash btn-icon btn-icon-secondary"></i>
+                                </script>
+                            </code>
+                        </pre>
+                </div>
+
+            </section>
+
+            <!-- Buttons with different sizes -->
+            <section class="component__variant">
+                <h3 class="variant__title">
+                    Different Sizes
+                </h3>
+                <p class="variant__desc">Depending on the use-case, you can select a small or a large button. Use <span
+                        class="highlight">btn-small and .btn-large</span> to change the size.</p>
+
+                <div class="bordered-container">
+                    <button class="btn btn-outline btn-small">Small</button>
+                    <button class="btn btn-outline btn-large">Large</button>
+                </div>
+
+                <div class="code__wrapper">
+                    <pre>
+                            <code class="language-markup">
+                                <script type="prism-html-markup">
+                                    <button class="btn btn-outline btn-small">Small</button>
+
+                                    <button class="btn btn-outline btn-large">Large</button>
+                                </script>
+                            </code>
+                    </pre>
+                </div>
+
+            </section>
+
+            <!-- Floating buttons -->
+            <section class="component__variant">
+
+                <h3 class="variant__title">
+                    Floating Action Buttons
+                </h3>
+                <p class="variant__desc">Floating buttons can be used to promote an <span class="highlight">important
+                        action </span>such as creating or sharing something. They are
+                    mostly used for <span class="highlight">positive actions</span> like adding something to
+                    wishlist.
+                    <ul>
+                        <li>Note that buttons can float against the parent container or the viewport. For static
+                            position, use class <span class="highlight">btn-floating.</span></li>
+                        <li>For floating with respect to the parent container, add one more class which specifies
+                            the position: <span class="highlight">top-left, top-right, bottom-left,
+                                bottom-right.</span></li>
+                        <li>For floating button with respect to viewport, add class <span
+                                class="highlight">float-on-screen.</span> For example, the button on bottom right of
+                            the screen.</li>
+                        <li>Position can be customized by overriding the positions specified in the respective
+                            class.</li>
```

Comment:
Docsite is well documented. Specified additional classes to the users which are highlighted also. Layout wise website is looking great.  Color scheme is awesome. Great work